documentation/book/assembly-custom-resources.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// getting-started.adoc
+
+[id='assembly-custom-resources-{context}']
+= Custom resources
+Custom Resource Definitions (CRDs) extend the Kubernetes API, providing definitions to create and add custom resources to an {ProductPlatformName} cluster. Custom resources are created as instances of CRDs.
+
+For {ProductName}, CRDs introduce custom resources specific to {ProductName} to an {ProductPlatformName} cluster, such as Kafka, Kafka Connect, Kafka Mirror Maker, and users and topics.  CRDs provide configuration instructions, defining the schemas used to instantiate and manage the resources specific to {ProductName}. CRDs also allow {ProductName} resources to benefit from native {ProductPlatformName} features like CLI accessibility and configuration validation.
```

Comment:
Suggested change:

"In {ProductName},"

Reply:
Suggested change:

"...and users and topics custom resources".

Reply:
Suggested change:

CRDs provide configuration instructions, defining the schemas used to instantiate and manage the {ProductName}-specific resources.